extensions/test/CognitoAuthentication.IntegrationTests/MfaAuthenticationTests.cs

```diff
@@ -110,7 +106,8 @@ public MfaAuthenticationTests()
             verifiedAttributes.Add(CognitoConstants.UserAttrPhoneNumber);
 
             //Create Role for MFA
-            using (var managementClient = new AmazonIdentityManagementServiceClient())
+            using (var managementClient = GetAmazonIdentityManagementServiceClient() 
+                )
```

Comment:
why is this in a new line?

Reply:
As .Net Core and .Net Xamarin use different constructor function of AmazonIdentityManagementServiceClient, we move the constructor statements to a GetAmazonIdentityManagementServiceClient and implement it in the partial class BaseAuthenticationTestClass

Reply:
I know why you made the change :)

Just asking why the `)` is in a new line.

Reply:
Not intentional. Already fixed it